trpcultivate_germplasm/tests/src/Functional/InstallTest.php

```diff
@@ -57,8 +78,12 @@ protected function setUp() :void {
     \Drupal::state()->set('is_a_test_environment', TRUE);
 
     // Open connection to Chado.
-    $this->connection = $this->getTestSchema(ChadoTestBrowserBase::PREPARE_TEST_CHADO);
+    $this->chado_connection = $this->getTestSchema(ChadoTestBrowserBase::PREPARE_TEST_CHADO);
 
+    // Set up our necessary materialized views.
```

Comment:
Lets make it clear why we do this and that it is hopefully temporary.

```suggestion
    // Set up our necessary materialized views.
    // This is needed because Tripal Core does not yet setup the Drupal side of
    // the test environment correctly when it comes to materialized views and
    // custom tables. Hopefully that will be resolved in the future.
```

Reply:
Added in commit https://github.com/TripalCultivate/TripalCultivate-Germplasm/pull/24/commits/5ccb7c80fdbc93fc264f01cf173197408d892959